packages/InteractiveProfilingTool-Tests.package/IPTTreeModelTestCase.class/instance/testGetTimeString.st

```diff
@@ -0,0 +1,9 @@
+tests
+testGetTimeString
+	
+	| timeStringActual |
+
+	timeStringActual := describedClass new getTimeString: tallyObject.
+
+	self assert: (timeStringActual includesSubString: '%').
+	self assert: (timeStringActual includesSubString: 'ms')
```

Comment:
To me this does not seem to test `getTimeString` well enough. Imagine a change to `getTimeString` that removes all time values (actual numbers) from the string. The test would be green when it should not.

Reply:
It is true that the regex test is probably covering more than this.
Our reasons for adopting this approach were:

1. Not all Squeak-Versions are capable of interpreting the regex, since this is not in the core but part of the vivide-package. This lead to be build failing on 5.0. We then had the choice to add the necessary methods to our project or use this version.
1. The regex is hard to understand and we figured a test should be easy to understand.

Reply:
see conversation below